dask/array/percentile.py

```diff
@@ -12,22 +12,29 @@
 
 @wraps(np.percentile)
 def _percentile(a, q, interpolation="linear"):
+    import pandas as pd
+
```

Comment:
Previously this pandas import was scoped within a `if a.dtype.name == "category":` check to avoid an `ImportError` when `pandas` isn't installed (for example, when Dask is installed with `dask[array]`). Am I missing something, or will we somehow avoid executing this code path when `pandas` isn't installed

Reply:
Ah crud, I thought this was still scoped in the correct way.  @galipremsagar can i ask you to fix ?

Reply:
Got it, but the reason the import was moved to the top of method call was to determine the timezone aware dtypes. The existing check for datetime type for those is going to fail: 
```
np.issubdtype(
        a.dtype, np.datetime64
    )
```

Reply:
A few lines above, we have a `if a.dtype.name == "category":` condition which, in part, serves as a "this is a pandas object, so it's safe to import pandas" check. Is there an equivalent check we can make here for timezone aware dtypes?

Reply:
Opened a PR to handle this: https://github.com/dask/dask/pull/8055